sentry.conf.py

```diff
@@ -259,11 +259,18 @@
 SENTRY_WEB_HOST = '0.0.0.0'
 SENTRY_WEB_PORT = 9000
 SENTRY_WEB_OPTIONS = {
+    'http': '%s:%s' % (SENTRY_WEB_HOST, SENTRY_WEB_PORT),
+    'protocol': 'uwsgi',
+    # This is need to prevent https://git.io/fj7Lw
+    'uwsgi-socket': None,
+    'http-keepalive': True,
+    'add-header': 'Connection: Keep-Alive',
+    'memory-report': False,
```

Comment:
What's this doing here?

Reply:
I figured we don't need memory reports on on-premise, do we?

Reply:
I don't see why we'd disable it though. tbh not entirely sure what it does.

Reply:
We explicitly set it to `true` in our default config and I have no idea why. Maybe we should reconsider that decision then. Will do in a follow-up diff.